ddtrace/_trace/tracer.py

```diff
@@ -775,7 +775,7 @@ def func_wrapper(*args, **kwargs):
                 )
 
             with self.trace(span_name, service=service, resource=resource, span_type=span_type):
-                yield from f(*args, **kwargs)
+                return yield from f(*args, **kwargs)
```

Comment:
Weirdly enough, this isn't valid syntax.

Reply:
```suggestion
                return_value = yield from f(*args, **kwargs)
                return return_value
```